packages/analytics-types/src/config.ts

```diff
@@ -92,7 +93,7 @@ export type BrowserOptions = Omit<
       transport: TransportType;
     }
   >,
-  'apiKey' | 'plugins'
+  'apiKey' | 'plugins' | 'sessionManager'
```

Comment:
hides sessionManager from public config interface - so they dont think they can pass this on init